tests/testthat/test-fims-rcpp.R

```diff
@@ -0,0 +1,19 @@
+test_that("Rcpp interface works for modules", {
+
+  fims <- Rcpp::Module("fims", PACKAGE = "FIMS")
+
+  expect_no_error(parameter <- new(fims$Parameter, .1))
+  expect_no_error(beverton_holt <- new(fims$BevertonHoltRecruitment))
+  expect_no_error(logistic_selectivity <- new(fims$LogisticSelectivity))
+  expect_no_error(ewaa_growth <- new(fims$EWAAgrowth))
+  logistic_selectivity$slope$value <- .7
+  logistic_selectivity$median$value <- 5.0
+
+  expect_equal(logistic_selectivity$slope$value, 0.7)
+  expect_equal(logistic_selectivity$get_id(), 1)
```

Comment:
question: how do we know that the get_id() is 1?

Reply:
@Andrea-Havron-NOAA explained to me that IDs for each type of module are assigned with FIMS, so the first module of its type is assigned 1. The next module of the same type would be assigned to, and so on.

This can be "resolved", but I'll leave the thread here now in case others want to see it.

Reply:
This does bring up the question for me if we should check recruitment and growth modules, too?

Reply:
I will add these additional tests, then will resolve this comment!